src/services/actionCableWebsocket.js

```diff
@@ -66,7 +67,7 @@ ngActionCable.factory("ActionCableWebsocket", ['$websocket', 'ActionCableControl
   };
   var new_data_stream = function(){
     if(dataStream === null) {
-      dataStream = $websocket(wsUrl);
+      dataStream = $websocket(wsUrl, null, null, wsOptions);
```

Comment:
it looks to me that this line should be: `dataStream = $websocket(wsUrl, null, wsOptions);`
because:  https://github.com/AngularClass/angular-websocket/blob/master/src/angular-websocket.js#L46

Or even `dataStream = $websocket(wsUrl, wsOptions);` 
because https://github.com/AngularClass/angular-websocket/blob/master/src/angular-websocket.js#L46-L50